webcompat/static/js/lib/ab.js

```diff
@@ -0,0 +1,11 @@
+/*global Mozilla experiment:true*/
+/*eslint no-undef: "error"*/
+
+var experiment = new Mozilla.TrafficCop({
+  id: "issue-form-experiment",
+  variations: {
+    "experiment=issueFormExperiment": 0
```

Comment:
0 means nobody will get the variation, correct?
(presumably we would change this to a meaningful value once we have the form variations built and ready to test)

Reply:
Reading https://github.com/mozilla/trafficcop#traffic-cop is helpful here.

Reply:
Yes, I initialized it with 0 just as a placeholder for the experiments.